### PR TITLE
RPC-113-check contract supports metaTransaction .

### PR DESCRIPTION
**Pull request by SKonstantinS**
Add only tests, which help to check contract supports metaTransaction

### CODE REVIEW

File: meta-tx/test/MetaTransaction.test.js [hunk 1]
```diff
@@ -78,13 +80,37 @@ const getTransactionData = async (nonce, abi, params) => {
...
+}
+
 contract("ERC721MetaTxTokenTestAllien", accounts => {
   let erc721NoMetaTx;
   let metaTxTest;
   let owner = accounts[0];
 
   beforeEach(async () => {
     metaTxTest = await deployProxy(MetaTxTest, ["MetaTxTest", "1"], { initializer: '__MetaTxTest_init' });
+    noMetaTxTest = await NoMetaTxTest.new();
```
Comment (disenotov) [inline meta-tx/test/MetaTransaction.test.js:113]: this contract needed in 1 test, so it's better to deploy it there only

File: meta-tx/test/MetaTransaction.test.js [hunk 2]
```diff
@@ -122,6 +148,30 @@ contract("ERC721MetaTxTokenTestAllien", accounts => {
...
+      functionSignature
+    } = await getTransactionData(nonce, getNonceAbi, [ZERO_ADDRESS]);
+    let resultExecMataTx = await metaTxTest.executeMetaTransaction(publicKey, functionSignature, r, s, v, {from: accounts[0]})
+
+    var newNonce = await metaTxTest.getNonce(publicKey);
+    assert.isTrue(newNonce.toNumber() == nonce + 1, "Nonce not incremented");
+  });
+
+  it("Check contract supports metaTransaction by method areMetaTxSupported, use contract yes MetaTx", async () => {
+  	let result = await areMetaTxSupported(metaTxTest);
```
Comment (disenotov) [inline meta-tx/test/MetaTransaction.test.js:166]: it would be cool to check if nonce is the same after areMetaTxSupported() executed

Comment (disenotov) [reply meta-tx/test/MetaTransaction.test.js:166]: also we could check areMetaTxSupported() for contract that doesn't have getNonce()

**Review by disenotov — CHANGES_REQUESTED** [review]

File: meta-tx/test/MetaTransaction.test.js (hunk 1 above)
Comment (SKonstantinS) [reply meta-tx/test/MetaTransaction.test.js:113]: ok, done

File: meta-tx/test/MetaTransaction.test.js (hunk 2 above)
Comment (SKonstantinS) [reply meta-tx/test/MetaTransaction.test.js:171]: ok, done

**Pull request merged**